spec/takeaway_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:order_klass) { double :order_klass, new: menu_klass, calculate_quantities: order, calculate_cost: order, calculate: order }
+  subject(:takeaway) { described_class.new(order_klass) }
+  let(:order) { double :order, inject: [["Dish 1" => 2.00]], length: 3, each: nil, calculate_cost: 2}
```

Comment:
Metrics/LineLength: Line is too long. [101/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)